File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/aboutgoods/RecommendedProducts.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.aboutgoods;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RecommendedProducts {

    @SerializedName("recomm_products")
    @Expose
    private RecommProducts recommProducts;
    @SerializedName("present")
    @Expose
    private int present;

    public RecommProducts getRecommProducts() {
        return recommProducts;
    }

    public void setRecommProducts(RecommProducts recommProducts) {
        this.recommProducts = recommProducts;
    }

    public int getPresent() {
        return present;
    }

    public void setPresent(int present) {
        this.present = present;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/aboutgoods/RecommendedProducts.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.aboutgoods

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RecommendedProducts {

    @SerializedName("recomm_products")
    @Expose
    var recommProducts: RecommProducts? = null
    @SerializedName("present")
    @Expose
    var present: Int = 0

}
